src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -183,6 +183,11 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
             {
                 AddSynthesizedAttribute(ref attributes, compilation.TrySynthesizeAttribute(WellKnownMember.System_ParamArrayAttribute__ctor));
             }
+
+            if (this.HasUnscopedRefAttribute && this.ContainingSymbol is SynthesizedDelegateInvokeMethod)
```

Comment:
>this.ContainingSymbol is SynthesizedDelegateInvokeMetho [](http://example.com/codeflow?start=48&length=55)

Consider unifying this check with the previous ```if```

Reply:
Left as is for now, but switched the order of the two `if` statements to avoid a conflict with the main branch.